README.markdown

```diff
@@ -31,7 +31,8 @@ This should give you the flexibility you need for customizing your input/output
 
 ### Building Your Phar
 
-Simply run `php package.php`, which will create the file: `build/doctrine-migrations.phar` for you.
+The PHAR is ready for [download here](http://github.com/downloads/ericclemmons/migrations/doctrine-migrations.phar), but if you still would like to build your PHAR
```

Comment:
@stof About the legacy system, not sure what to replace it with but there must be another place where these PHARs could be hosted on. About the need for PHARs, i see them useful if you are introducing components in your project but are not ready to use a framework console yet, i.e. no symfony console exists